module/apmgopg/hook.go

```diff
@@ -0,0 +1,72 @@
+// ownership. Elasticsearch B.V. licenses this file to you under
+// the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+// +build go1.9
+
+package apmgopg
+
+import (
+	"github.com/go-pg/pg"
+
+	"go.elastic.co/apm"
+	"go.elastic.co/apm/module/apmsql"
+)
+
+const elasticApmSpanKey = "go-apm-agent:span"
+
+type QueryHook struct {
+	Database string
+	Username string
+
+	UseFormattedQuery bool
```

Comment:
I'm not familiar with go-pg, but I believe this will include the query parameters in the SQL. Is that correct? Could you add a little doc comment explaining the difference?

Reply:
To be honest I'm not even sure I want to expose this since it contains the parameters it's a security hole

Reply:
If you don't need it, I'm cool with leaving it out.